README.adoc

```diff
@@ -93,7 +93,8 @@ application. Did you notice that there is not a single line of XML? There is no
 file, either. This web application is 100% pure Java and you did not have to deal with
 configuring any plumbing or infrastructure. Spring Boot handles all of that for you.
 
-Logging output is displayed. The service should be up and running within a few seconds.
+Go ahead and run `main()`. Logging output will be displayed and the service should be up 
+and running within a few seconds.
```

Comment:
Remove "and "

Reply:
Which "and" (they all look necessary to me)?

Reply:
The two lines should be:

Run `main()`. Logging output is displayed and the service should be
running within a few seconds.

Github makes me comment on one line at a time, so the "and" in question was the "and" at the beginning of line 37 (the only "and" in that line.

Reply:
Got it; all fixed.